src/components/Card/Card.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import classes from './Card.module.scss';
+
+interface CardProps {
+  title?: string;
+  children: JSX.Element | JSX.Element[];
+}
+
+const Card: React.FC<CardProps> = ({ title, children }) => {
+  return (
+    <div className={`${classes['c-card']}`}>
+      <h1 className={`${classes['c-card__title']}`}>{title}</h1>
```

Comment:
What happens if we don't pass title here since according to props it is possible?